test/test_verilog_verilator_search_path.vader

```diff
@@ -0,0 +1,27 @@
+Before:
```

Comment:
See `:help ale-dev` for information on writing tests for linter definitions. Try to structure this more like other linter tests, such as `test/command_callback/test_iverilog_command_callback.vader`,

Reply:
Should have read the doc earlier...